tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/SerilogTests.cs

```diff
@@ -14,40 +14,20 @@ namespace Datadog.Trace.ClrProfiler.IntegrationTests
 {
     public class SerilogTests : LogsInjectionTestBase
     {
-        private readonly LogFileTest[] _log200FileTests =
-            {
-                new LogFileTest()
-                {
-                    FileName = "log-textFile.log",
-                    RegexFormat = @"{0}: {1}",
-                    UnTracedLogTypes = UnTracedLogTypes.EmptyProperties,
-                    PropertiesUseSerilogNaming = true
-                },
-                new LogFileTest()
-                {
-                    FileName = "log-jsonFile.log",
-                    RegexFormat = @"""{0}"":{1}",
-                    UnTracedLogTypes = UnTracedLogTypes.None,
-                    PropertiesUseSerilogNaming = true
-                }
-            };
-
-        private readonly LogFileTest[] _logPre200FileTests =
-        {
+        private readonly LogFileTest _txtFile =
             new LogFileTest()
             {
                 FileName = "log-textFile.log",
                 RegexFormat = @"{0}: {1}",
-                TracedLogTypes = TracedLogTypes.NotCorrelated,
                 UnTracedLogTypes = UnTracedLogTypes.EmptyProperties,
                 PropertiesUseSerilogNaming = true
-            }
-        };
+            };
 
         public SerilogTests(ITestOutputHelper output)
             : base(output, "LogsInjection.Serilog")
         {
             SetServiceVersion("1.0.0");
+            EnableDebugMode();
```

Comment:
Do we need this or was it for local testing?

Reply:
No this isn't needed for the test case. Removed in 220bd64a22385aa2120912bad4b922c35a8e8de7